tests/e2e/specs/gutenberg-plugin.test.js

```diff
@@ -1,26 +1,48 @@
-import {
-	activatePlugin,
-	deactivatePlugin,
-	installPlugin,
-	uninstallPlugin,
-} from '@wordpress/e2e-test-utils';
-
-describe( 'Gutenberg plugin', () => {
-	beforeAll( async () => {
-		await installPlugin( 'gutenberg' );
-	} );
+/**
+ * WordPress dependencies
+ */
+import { test, expect } from '@wordpress/e2e-test-utils-playwright';
+
+test.describe( 'Gutenberg plugin', () => {
+	// Increasing timeout to 5 minutes because potential plugin install could take longer.
+	test.setTimeout( 300_000 );
+
+	test.beforeAll( async ( { requestUtils } ) => {
+		// Install Gutenberg plugin if it's not yet installed.
+		const pluginsMap = await requestUtils.getPluginsMap();
+		if ( ! pluginsMap.gutenberg ) {
+			await requestUtils.rest( {
+				method: 'POST',
+				path: 'wp/v2/plugins?slug=gutenberg',
+			} );
+		}
 
-	afterAll( async () => {
-		await uninstallPlugin( 'gutenberg' );
+		// Refetch installed plugin details. It avoids stale values when the test installs the plugin.
+		await requestUtils.getPluginsMap( /* forceRefetch */ true );
```

Comment:
This prevents tests from failing with `Error: The plugin "gutenberg" isn't installed` when installing the plugin.

Reply:
Right, because `deactivateMap` calls `getPluginsMap()`. Makes sense 👍